159 the list of minor system bugs

mehmeteyupoglu opened the pull request.

mehmeteyupoglu committed:
feat: add navigation to community details on community name click

mehmeteyupoglu committed:
refactor: remove unused imports in PostView component

mehmeteyupoglu committed:
feat: add navigation to home on logo click in LogoAsText component

mehmeteyupoglu committed:
fix: add optional chaining to content mapping in DisplayPost component

mehmeteyupoglu committed:
fix: update heading from 'Hot Topics' to 'Feed' in Home component

mehmeteyupoglu committed:
feat: enhance navigation by adding click handlers to title and tags in DisplayPost component

mehmeteyupoglu committed:
fix: remove external link icon and add navigation on community card click in IndividualCommunity component

mehmeteyupoglu committed:
feat: add number_of_posts field to community data and update components to display post count

mehmeteyupoglu committed:
feat: improve field name display in DisplayPost component with formatting and conditional styling

mehmeteyupoglu committed:
feat: add notification and navigation on successful post submission in MapFields component

mehmeteyupoglu committed:
Merge branch 'dev' into 159-the-list-of-minor-system-bugs

mehmeteyupoglu committed:
feat: update text color for dark mode in Copyright and RecentCommunity components

mehmeteyupoglu committed:
feat: add dropdown menu for post actions in DisplayPost component

mehmeteyupoglu committed:
feat: enhance DisplayPost and RenderField components with improved dropdown placement and image styling

mehmeteyupoglu committed:
feat: integrate dropdown menu for comment actions in Comment component

mehmeteyupoglu committed:
feat: implement "See more" functionality for comment content display

mehmeteyupoglu committed:
feat: update image source handling in RenderField component

mehmeteyupoglu committed:
feat: change default tab to 'posts' in CommunityDetail component and reorganize tab menu

mehmeteyupoglu committed:
feat: add delete template functionality in CommunityService and Template component

Burak-Cosar commented:
@mehmeteyupoglu I had a quick look, and noticed below:

- [x] Comment options buttons fall behind other UI elements:
<img width="299" alt="Screenshot 2024-11-28 at 21 41 57" src="https://github.com/user-attachments/assets/7ec94717-1f2b-47b8-9c33-049ffc0ee43a">

- [x] When **not inside the post** , like button is not filled after liking (I liked both posts below):
<img width="278" alt="Screenshot 2024-11-28 at 21 44 17" src="https://github.com/user-attachments/assets/f438fa1f-6e19-4c0b-ac6c-39d9105f797c">

- [x] In the Posts tab of community, it would be nice to have a button to "Post"

- [x] In Posts, we can move "Delete" button inside the Options dropdown (as in Comments):
<img width="467" alt="Screenshot 2024-11-28 at 21 53 37" src="https://github.com/user-attachments/assets/0193403d-f902-4797-8f57-edbd3f8c9c4c">

mehmeteyupoglu committed:
feat: enhance dropdown menu z-index and refactor delete action to dropdown in DisplayPost component

mehmeteyupoglu committed:
feat: enhance posts endpoint to include user likes and tags in response

mehmeteyupoglu committed:
feat: refactor delete action in DisplayPost component to disable for non-owners

mehmeteyupoglu committed:
feat: add 'Post' button in CommunityDetail component for members and adjust tab list padding

mehmeteyupoglu committed:
feat: update layout of footer in DisplayPost component for improved responsiveness

Burak-Cosar reviewed (changes requested):
One final thing, when inside Report View, the button texts are white, thus invisible:
![Screenshot 2024-11-30 at 11 30 41](https://github.com/user-attachments/assets/b534f47b-c695-4093-b69c-3afe78c2c54b)

Commit:
- Fix Report View Button Text Colors
- Fix Report Reports Table
- Move Post button place from all tabs to only Posts tab

GaitovRK reviewed (approved):
All good!
